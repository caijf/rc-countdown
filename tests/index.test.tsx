import React from 'react';
import renderer from 'react-test-renderer';
import CountDown from '../src';

describe('CountDown', () => {
  beforeEach(() => {
    jest.useFakeTimers();
  });

  it('should de defined', () => {
    expect(CountDown).toBeDefined();
  });

  it('snapshot', async () => {
    const component = renderer.create(<CountDown time={60 * 1000} className='test' autoStart={false} />);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});