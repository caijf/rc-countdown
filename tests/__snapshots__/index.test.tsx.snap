// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountDown snapshot 1`] = `
<span
  className="test"
>
  00:01:00
</span>
`;
